docker-compose.yaml

```diff
@@ -0,0 +1,78 @@
+name: photoview
+
+volumes:
+  mariadb:
+  postgres:
+
+services:
+  mysql:
+    image: mariadb:11.4
```

Comment:
Why (here and in PGSQL service) do you prefer to hardcode the particular version instead of using `lts` and `16-alpine` for PGSQL, like in the PROD example? We might end up with not matching DB versions quickly.

Reply:
I don't have a strong idea to pin the database version.

What do you think about new features in the future database?

Reply:
I'd propose reflecting the PROD compose in the topic of dependencies versioning - in this way we ensure that the development environment is as close to PROD as possible, so all DB-related behaviors would be the same

As currently we don't utilize any specific features of DBMS, as well as I'm not aware of any plans to utilize some edge features, I'd propose to stick to the latest stable versions (LTS for MariaDB and <major release>-alpine for PGSQL) - they are stable enough to rely on them, while provide the latest and well-tested fixes and improvements.

Reply:
Done.